doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -119,92 +107,43 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-* Added:
-
-  * :ref:`peripheral_ams_client` sample.
+* Added :ref:`peripheral_ams_client` sample.
 
 * Updated:
 
   * :ref:`direct_test_mode` sample:
 
+    * Updated:
+
+      * The front-end module test parameters are not set to their default value after the DTM reset command.
```

Comment:
This entry sounds a bit weird alone, not to mention with "Updated" in front of them... Maybe:
```suggestion
      * The front-end module test parameters are not set to their default value after the DTM reset command anymore.
```

Reply:
Updated